.eleventy.js

```diff
@@ -1,12 +1,21 @@
 const fs = require("fs");
 const inclusiveLangPlugin = require("@11ty/eleventy-plugin-inclusive-language");
+const format = require("date-fns/format");
 
 module.exports = eleventyConfig => {
   eleventyConfig.addPassthroughCopy("src/css");
   eleventyConfig.addPassthroughCopy("src/img");
   eleventyConfig.addPassthroughCopy("src/js");
   eleventyConfig.addPassthroughCopy("src/favicon.ico");
 
+  // Filters
+  eleventyConfig.addNunjucksFilter("formatDate", (startedAt, endedAt) => {
```

Comment:
Does this work as a universal filter? `eleventyConfig.addFilter`

Reply:
Yes it does, that name sounds good to me.. so I've just pushed this change 😁